### DISCUSSION TITLE
Control frigate from HA

### DISCUSSION THREAD

**Discussion by mpfl** [blakeblackshear/frigate#2297] [Ask A Question]
Is it possible to control Frigate's behaviour, such as enabling and disabling recording, from Home Assistant?

I have a camera in my house to keep an eye on my dog when I'm not home.

I'd like to disable recording on that camera when I'm home so I don't fill my drive up with video of my butt.

**Comment by transplier** [comment]
Recordings, detection, and snapshots are controllable per-camera via MQTT. The HA addon exposes switches for this (make sure the addon is up to date).
